packages/core/src/utils/utils.ts

```diff
@@ -432,3 +432,18 @@ export const colorSchemeCustomForeground = ({ colorScheme, color, iconUsesColorS
 export const textTransform = (props) => (props.textTransform ? { textTransform: props.textTransform } : null)
 
 export const textWrap = (props) => (props.textWrap ? { textWrap: props.textWrap } : null)
+
+export function debounce(func, wait, immediate) {
+  let timeout
+  return function () {
```

Comment:
Don't you need to spread the arguments into this function? I had assumed that this creates a new scope for the `args` setting below and comes up empty.